monofony/fixtures-plugin/0.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Monofony\\Plugin\\FixturesPlugin\\MonofonyFixturesPlugin": ["all"]
```

Comment:
I dont understand this bundle either. 

- It looks like a library because it is not using any "bundle features" like config or service definitions. 
- It got missing dependencies in composer.json
- It contains 2 interfaces and one abstract class. 

So what does this package do?

Reply:
This package adds the Sylius Fixtures bundle with two abstract classes from Sylius Core Bundle (which depends on too much things to use it directly).
It also contains two data fixtures for the 2 first entities configured with this project (administrators and customers/users).
But you're right, I can create a library instead of a bundle too.

Reply:
@Nyholm @lchrusciel
I think it could be renamed to "monofony/fixtures-pack" cause it contains only composer dependencies and configuration files for the 2 default resources. 

@Nyholm FYI I opened a pull request on Sylius fixtures bundle to include these 2 abstract classes directly on the Sylius package. https://github.com/Sylius/SyliusFixturesBundle/pull/29